webapp/pages/map.spec.js

```diff
@@ -4,6 +4,11 @@ import { mount } from '@vue/test-utils'
 import VueMeta from 'vue-meta'
 import Vuex from 'vuex'
 import Map from './map'
+import MapboxGeocoder from '@mapbox/mapbox-gl-geocoder'
+
+jest.mock('@mapbox/mapbox-gl-geocoder', () => {
```

Comment:
how is this compatible with `'@mapbox/mapbox-gl-geocoder': 'identity-obj-proxy',` in the jest config? Do we still need this line now?

Reply:
```
import idObj from 'identity-obj-proxy';
console.log(idObj.foo); // 'foo'
console.log(idObj.bar); // 'bar'
console.log(idObj[1]); // '1'
```
So there is no constructor anymore. I have to mock/recreate the constructor